engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/runtime/ContextIsEnabledNode.java

```diff
@@ -0,0 +1,32 @@
+package org.enso.interpreter.node.expression.builtin.runtime;
+
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.node.expression.builtin.text.util.ExpectStringNode;
+import org.enso.interpreter.runtime.EnsoContext;
+import org.enso.interpreter.runtime.callable.atom.Atom;
+import org.enso.interpreter.runtime.error.PanicException;
+import org.enso.interpreter.runtime.state.ExecutionEnvironment;
+import org.enso.interpreter.runtime.state.State;
+
+@BuiltinMethod(
+    type = "Context",
+    name = "is_enabled",
+    description = "Check if the context is enabled in the provided execution environment.")
+public class ContextIsEnabledNode extends Node {
+  private @Child ExpectStringNode expectStringNode = ExpectStringNode.build();
+
+  Object execute(State state, Atom self, Object environmentName) {
+    String envName = expectStringNode.execute(environmentName);
+    ExecutionEnvironment currentEnv = state.currentEnvironment();
+    if (!currentEnv.getName().equals(envName)) {
+      Atom error =
+          EnsoContext.get(this)
+              .getBuiltins()
+              .error()
+              .makeUnimplemnted("execution environment mismatch");
+      throw new PanicException(error, this);
```

Comment:
I really doubt a `Panic` is what we want! I don't even understand what's the purpose of specifying environment name in the `is_enabled` check. Environments are supposed to be a set of contexts. All the user wants is to take a context and check if it is currently enabled. E.g. `Context.Input.is_enabled`.

Reply:
This isn't really specified well in the design doc. Nor is the usage of the environment name so if you feel strongly about it, probably worth bringing up with that doc authors.

Reply:
> At least for the moment - there is no intention to specify anything other than the current context when calling this function.

Obviously! Specifying anything else makes no sense. At least I cannot envision the use-case.

I'd say: One doesn't care about the environment. One only wants to find out if a `Context` `is_enabled` "right now". Nothing else makes sense.